frontend/src/components/Samples.vue

```diff
@@ -112,7 +112,10 @@
       <tr v-for="entry in subsetEntries" :key="entry._id">
         <td v-if="entry.rowspan.short_name > 0" :rowspan="entry.rowspan.short_name">{{entry.short_name}}</td>
         <td class="dataset-column" v-if="entry.rowspan.dataset > 0" :rowspan="entry.rowspan.dataset">
-          <a :href="'https://cms-pdmv-prod.web.cern.ch/mcm/requests?dataset_name=' + entry.dataset" target="_blank">{{entry.dataset}}</a>
+          {{entry.dataset}}: 
```

Comment:
This will look nicer with long dataset names:
```suggestion
          {{entry.dataset}}
          <br>
```